examples/cli/crud_partition.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+echo "CRUD partition script."
+HMC_HOST="Put here the HMC host"
+HMC_USER="Put here HMC user"
+CPC_NAME="Put here the name of the CPC"
+PART_NAME="TESTPARTITION"
+
+eval $(zhmc -h $HMC_HOST -u $HMC_USER session create)
+zhmc cpc list
+zhmc partition list $CPC_NAME
```

Comment:
Good idea :-) I like the jq tool but I think the python solution is good enough and I agree it avoids a new dependency.